pkgs/misc/emulators/citra/default.nix

```diff
@@ -2,13 +2,13 @@
 
 mkDerivation {
   pname = "citra";
-  version = "2020-12-07";
+  version = "2021-06-22";
 
   # Submodules
   src = fetchgit {
```

Comment:
Can you convert this to fetchFromGitHub?

Reply:
I tried this, but the citra repo fetches submodules from other locations, and I'm not 100% sure how to get fetchFromGitHub to fetch submodules.

Reply:
https://github.com/NixOS/nixpkgs/blob/6ecbfcdb36d5b6b1daee265d3e2efad0be8ce21f/doc/builders/fetchers.chapter.md#L33

Reply:
Awesome, sorry about that I couldn't find the fetchers docs.